.github/workflows/on-pull-request-opened-and-synchronize.yml

```diff
@@ -208,7 +208,7 @@ jobs:
           pattern: visual-test-report-blob-*
       - name: Merge blob reports
         run: |
-          PLAYWRIGHT_HTML_ATTACHMENTS_BASE_URL=https://${{ vars.CLOUDFLARE_CUSTOM_DOMAIN }}/${{ github.head_ref }}/visual-test-report/data/ pnpm exec playwright merge-reports --reporter html ./visual-test-report-blob
```

Comment:
`github.head_ref` is empty when a PR enters the merge queue. I believe `github.event.pull_request.head.ref` gives us a reference to the original branch. Fingers crossed. There may be some trial and error on this one.

Reply:
Same deal, [unfortunately](https://github.com/CrowdStrike/glide-core/actions/runs/14626075366/job/41037828563). Still deploying to `/`. I'll do some more research and revisit tomorrow.